renderer/pyscript-offline/public/README.md

```diff
@@ -0,0 +1,19 @@
+# UNLWS Renderer
+A tool to specify UNLWS texts in code and display them.
+
+## Setup
+
+1. Clone the repo
+2. Open a terminal in `...\unlws\renderer\pyscript-offline\public`
+3. Start a local web server. If you have Python installed, this can be done by running
+    ```sh
+    python3 -m http.server
```

Comment:
Change to "./run_local_server.sh" 😛

Reply:
I added another check at the start to try using "py" before trying "python3" or "python", because Windows sends those commands (if they're used alone) to the MS Store to install python, and so they're identified as valid commands. (if used with arguments they'll still give an error). I imagine this addition won't conflict with anything.

Now it should work for anyone that has python, and should give a proper message for non-Windows users who don't have it. Windows users without python might not always get clear directions.